lexer_schemas/common.py

```diff
@@ -18,6 +18,11 @@ def wrapped(cls):
     return wrapped
 
 
+class Channel(str, Enum):
```

Comment:
In the inventory context, does this need to match to the internal version? https://github.com/lexerdev/customer-api/blob/master/customer_api/records/entity.py#L64-L67

Reply:
Yeah, my plan was to convert the value in the CDE https://github.com/lexerdev/customer-api/pull/1165